src/libs/kata-sys-util/src/protection.rs

```diff
@@ -12,21 +12,22 @@ use std::fmt;
 use std::path::Path;
 use std::path::PathBuf;
 use thiserror::Error;
+use serde::{Deserialize, Serialize};
```

Comment:
Could lines 12 and 13 be combined as "use std::path::{Path, PathBuf}?

Reply:
@mkbhanda Yes it definitely could. But since that is an unrelated change I prefer not to include it here, maybe in a separate change.